corral.json

```diff
@@ -1,9 +1,21 @@
 {
+  "packages": [],
   "deps": [
     {
       "locator": "ponylang/ponycheck",
       "version": "0.5.4"
+    },
+    {
+      "locator": "github.com/ponylang/ponycheck.git",
+      "version": ""
```

Comment:
can you pin us to `0.5.4` here?

Reply:
I think CI is also failing because this is missing.

If you just added this via `corral` CLI i think we should create an issue there, is also generated an empty info section below, which shouldnt be there just from the `corral` invocation.